src/main/java/org/dcsa/bkg/repositories/CarrierClauseRepository.java

```diff
@@ -0,0 +1,9 @@
+package org.dcsa.bkg.repositories;
+
+import org.dcsa.bkg.model.transferobjects.CarrierClauseTO;
+import org.dcsa.core.repository.ExtendedRepository;
+
+import java.util.UUID;
+
+public interface CarrierClauseRepository extends ExtendedRepository<CarrierClauseTO, UUID> {

```

Comment:
Move to event-core.

Reply:
https://github.com/dcsaorg/DCSA-Event-Core/pull/142